x-pack/solutions/security/plugins/security_solution/public/data_view_manager/redux/listeners/init_listener.ts

```diff
@@ -72,14 +84,25 @@ export const createInitListener = (
           alertDataView.title
         );
 
+        logger.info(`Explore Data View created: 
+          - Explore Data View: ${exploreDataView.title} (ID: ${exploreDataView.id})`);
+
+        // Store the created data views in the Redux state
         listenerApi.dispatch(sharedDataViewManagerSlice.actions.addDataView(exploreDataView));
 
         // NOTE: This is later used in the data view manager drop-down selector
         const dataViews = await dependencies.dataViews.getAllDataViewLazy();
+
+        logger.info(`Fetched ${dataViews.length} data views from the Data Views service.`);
+
         const dataViewSpecs = await Promise.all(dataViews.map((dataView) => dataView.toSpec()));
 
+        logger.info(`Converted data views to specs.`);
+
         listenerApi.dispatch(sharedDataViewManagerSlice.actions.setDataViews(dataViewSpecs));
 
+        logger.info(`Set data views in the Redux state.`);
```

Comment:
Same here, we could log how many data views we are setting to Redux, and maybe even their names or something like that?